draft-ietf-scitt-architecture.md

```diff
@@ -218,15 +218,11 @@ Equivocation:
 
 : a state where it is possible for a Transparency Service to provide different views of its Append-only log to Relying Parties about the same Artifact {{EQUIVOCATION}}.
 
-Feed:
-
-: A collection of Receipts, as recorded by the Transparency Service, based on filtering of properties from the envelope including, but not limited to the `sub` field of the `CWT_Claims`.
-Relying Parties may use the Feed to ensure completeness and Non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the Artifact they are evaluating.
-
 Issuer:
 
-: organizations, stakeholders, and users involved in creating or attesting to supply chain artifacts, releasing authentic Statements to a definable set of peers.
+: an identifier that represents the organization, device, user or entity responsible for securing Statements about supply chain Artifacts and Receipts.
```

Comment:
replacing "attesting" with "securing" seems a bit confusing as what does "securing mean". Issuers make statements about artifacts. That is synonymous with Issuers attest...
```suggestion
: an identifier representing an organization, device, user, or entity attesting to Statements about supply chain Artifacts.
```

Reply:
I don't think we should us the word "attest"

Reply:
I reverted 'attesting to' to 'securing'. @SteveLasker, that good enough?